Makefile

```diff
@@ -302,7 +302,10 @@ endif
 	@sed -i -e 's/Json/JSON/' $@ && rm -f ./-e
 
 .PHONY: cross
-cross: minikube-linux-amd64 minikube-linux-arm64 minikube-darwin-amd64 minikube-windows-amd64.exe ## Build minikube for all platform
+cross: minikube-linux-amd64 minikube-darwin-amd64 minikube-windows-amd64.exe ## Build minikube for all platform
+
+.PHONY: exotic
```

Comment:
add ## help text what exotic is to the new contributor

Reply:
Sure thing, basically here it means non-amd64 (i.e. non-standard)

Reply:
once ## comment added for this we can merge this

Reply:
Thought I added it ?